docs/get-started/enterprise-and-community-edition.md

```diff
@@ -87,7 +87,7 @@ Ant Media provides _free Enterprise Licenses_\* for the \*students, academics, a
 
 ## Supported Environments
 
-Ant Media Server runs on **Linux(Ubuntu)** and **MacOS**. it supports only x64 architecture. Ubuntu 18.04 is officially supported and auxiliary scripts are provided for Ubuntu 20.04 and CentOS 8. In addition, It's known that Ant Media Server is used on SUSE, Debian, RHLE distributions as well.
+Ant Media Server runs on **Linux(Ubuntu)** and **MacOS**. it supports both x64 and ARM architecture. Ubuntu 20.04, Ubuntu 22.04 and CentOS 8 are officially supported. In addition, It's known that Ant Media Server is used on SUSE, Debian, WSL, and RHLE distributions as well.
```

Comment:
This doesn't read well (also lose the capital I in "It's): "In addition, It's known that Ant Media Server is used on SUSE, Debian, WSL, and RHLE distributions as well." 

Maybe something along the lines of "Ant Media Server has also been successfully installed on SUSE, Debian, WSL and RHLE but are not officially supported.

Also, do we want to mention any other distributions if we are not 100% sure they will work?

Reply:
The idea behind this PR is to mainly structure the documentation flow like keeping right subsection under different sections.
And then all of us has been assigned tasks to look into specific sections and topics to review and make corrections and changes wherever needed.